### PR TITLE
PR chnages to new repo and new repo changes

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/tests/e2e/README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@
 10. Now, run the following commands to perform the e2e test execution.
 	
 	```
-	cd $GOPATH/src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e
+	cd $GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/README.md]: This should be `src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher`.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/conf/prod-us-east.conf [hunk 2]
```diff
@@ -41,18 +41,18 @@ tor01_PVG_CRUISER_PRIVATE_VLAN=2259903
 
 # For plugin build
 GOPATH=$GOPATH  					# Read from environment
-BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/
+BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/conf/prod-us-east.conf [hunk 3]
```diff
@@ -41,18 +41,18 @@ tor01_PVG_CRUISER_PRIVATE_VLAN=2259903
...
 USER_NAMESPACE=blockvolume
 PLUGIN_IMAGE=armada-block-volume-attacher
-PLUGIN_BUILD=latest
+PLUGIN_BUILD=$PLUGIN_BUILD
 
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
-HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/block-volume-attacher/helm/ibmcloud-block-storage-attacher/
+HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/helm/ibmcloud-block-storage-attacher/
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/conf/stage.conf [hunk 4]
```diff
@@ -38,20 +38,18 @@ sao01_PVG_CRUISER_PRIVATE_VLAN=2274515
 
 # For plugin build
 GOPATH=$GOPATH						# Read from environment
-BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/armada-storage-block-plugin
+BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/stage.conf]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/conf/stage.conf [hunk 5]
```diff
@@ -38,20 +38,18 @@ sao01_PVG_CRUISER_PRIVATE_VLAN=2274515
...
-PLUGIN_BUILD=1.7
-DRIVER_BUILD=1.7
+PLUGIN_IMAGE=ibmcloud-block-storage-attacher
+PLUGIN_BUILD=$PLUGIN_BUILD
 
 # Helm chart install
 KUBECONFIG=$KUBECONFIG
 HELM_CHART_REPO=$HELM_CHART_REPO
-HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/armada-storage-block-plugin/deploy/public-helm-charts/stable/ibmcloud-block-storage-plugin
+HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/helm/ibmcloud-block-storage-attacher/
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/stage.conf]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go [hunk 6]
```diff
@@ -33,11 +33,11 @@ var (
 	clusterName   = ""
 	pvfilepath    = ""
 	pv            *v1.PersistentVolume
-	e2epath       = "src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/e2e-tests/"
+	e2epath       = "src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/e2e-tests/"
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go [hunk 7]
```diff
@@ -12,13 +12,13 @@
...
-	. "github.com/onsi/gomega"
 	"bufio"
 	"bytes"
-	"fmt"
-	"github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/framework"
 	"errors"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/framework"
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/e2e-tests/e2e_suite_test.go [hunk 8]
```diff
@@ -14,7 +14,7 @@ package e2e
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/framework"
+	"github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/framework"
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/e2e-tests/e2e_suite_test.go]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/e2e.sh [hunk 9]
```diff
@@ -9,6 +9,13 @@ set +a
 # check mandatory variables
 [ -z "$GOPATH" ] && echo "Need GOPATH for plugin build and test executions(e.g export GOPATH=\path\to)" && exit 1
 
+E2E_PATH="src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/e2e-tests/"
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/e2e.sh]: This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.

File: block-storage-attacher/tests/e2e/conf/prod-us-east.conf [hunk 10]
```diff
@@ -67,5 +67,10 @@ TEST_CODE_BUILD=true
 TEST_SUITE_RUN=basic
 
 # Testcase variables
-PLUGIN_NAME=ibm.io/ibmc-block
+PLUGIN_NAME=ibm.io/ibmc-blockvolume-attacher
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: Need to check where is this variable used and then confirm if value is correct.

File: block-storage-attacher/tests/e2e/conf/prod-us-east.conf [hunk 11]
```diff
@@ -41,18 +41,18 @@ tor01_PVG_CRUISER_PRIVATE_VLAN=2259903
 
 # For plugin build
 GOPATH=$GOPATH  					# Read from environment
-BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/
+BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/
 
 # Latest Storage Plugin images to be used for e2e testing
 IMAGE_REGISTRY=registry.ng.bluemix.net
 USER_NAMESPACE=blockvolume
 PLUGIN_IMAGE=armada-block-volume-attacher
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: PLUGIN_IMAGE has to be changed as per stage.conf

File: block-storage-attacher/tests/e2e/conf/prod-us-east.conf (hunk 10 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: Here below we use this value 
block-storage-attacher/tests/e2e/framework/framework.go:        pluginName := os.Getenv("PLUGIN_NAME")
block-storage-attacher/tests/e2e/common/common_util.go: pluginNameEnv := os.Getenv("PLUGIN_NAME")

To print the plugin name while publishing the result

Comment (akgunjal) [reply block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: Use `ibm.io/ibmc-blockattacher` as PLUGIN_NAME

**Pull request merged**